packages/generator/src/vdm-types.ts

```diff
@@ -58,7 +61,13 @@ export interface VdmNavigationProperty extends VdmPropertyDescriptor {
   from: string;
   to: string;
   toEntityClassName: string;
-  multiplicity: string;
+  /**
+   * @deprecated Since v1.21.3. Will not be replaced.
+   */
+  multiplicity?: string;
+  /**
+   * @deprecated Since v1.21.3. Use `isCollection` instead.
```

Comment:
In this interface I do not see a `isCollection` what is this referring to?

Reply:
It is in the `PropertyDescriptor`.

Reply:
yes, sorry.